README.md

```diff
@@ -103,6 +103,25 @@ Currently the Cld2 bindings are only generated for `linux-x86-64`, so if your
 machine is different it probably won't work. In such a case, just execute it
 with the `java_only` profile.
 
+Launch service using docker
+--------------------------
+The service can be also launched using docker. It requires the installation of `docker` and `docker-compose`. See [docker home page](https://docs.docker.com/) for installation references.
+
+Once docker is installed execute
+```bash
+$ docker-compose up -d
+```
+this will build the image of the service and launch it. To see on which port the service is exposed,
+execute `docker-compose ps`. This will display a table similar to the example below:
+```bash
+$ docker-compose ps
+    Name                   Command               State                        Ports
+---------------------------------------------------------------------------------------------------------
+weslang_api_1   java -jar buck-out/gen/jav ...   Up      0.0.0.0:32774->8080/tcp, 0.0.0.0:32775->9001/tcp
+```
+From example above, the service on port `8080` is exposed on port `32774` and service exposed on port `9001` is
+exposed on port `32775`.
```

Comment:
Add a line explaining that the health endpoint running on port 9001 can be accessed by executing the following:
`docker exec CONTAINER_ID python -c "import urllib; print urllib.urlopen('http://127.0.0.1:9001/metrics').read();" | python -m json.tool`

And the container id can be get by executing `docker ps`